src/importer/defaults/rules/convertIcons.js

```diff
@@ -19,6 +31,10 @@ export default function convertIcons(main, document) {
       if (name) {
         span.innerHTML = `:${name}:`;
         img.replaceWith(span);
+
+        if (callback && typeof callback === 'function') {
+          callback(src);
```

Comment:
Shouldn't you pass the name also ?
```suggestion
          callback(src, name);
```

Reply:
That might be helpful as well, to create a mapping of original to new value.